grafana.sh

```diff
@@ -263,7 +263,7 @@ set_vcap_datasource_prometheus() {
 	# list of datasources to insert/update depending
 	# what's available in the database
 	datasources:
-	- name: ${name}
+	- name: ${name} Prometheus
```

Comment:
Is this correct?

Reply:
Yes, this is right. 

Before, the name was just the name of the service instance, e.g. wenxin-new-new-prometheus-test.

Now that we have multiple datasources, I want to make the names more specific, and differentiate Prometheus from Alertmanager.

So now there'll be two datasources, called:
wenxin-new-new-prometheus-test Prometheus
wenxin-new-new-prometheus-test AlertManager